### DISCUSSION TITLE
[Saran & Ide] Fitur Kirim File / Gambar Dengan Base64 Encode

### DISCUSSION THREAD

**Discussion by kotak47** [kirimwa-id/forum#109] [Saran dan Ide]
Terimakasih sebelumnya mas Rio, API Wa nya berjalan normal. Sangat membantu untuk reminder2 sederhana saya.

Saya ada ide fitur agar memungkinkan mengirim File maupun Image menggunakan Base64. Sehingga dapat mengirim file/gambar tanpa harus upload ke Hosting / Public Server.

Salam,

**Comment by rioastamal** [comment]
@kotak47, terima kasih atas idenya.
